mmv1/third_party/terraform/services/compute/resource_compute_backend_service_test.go.erb

```diff
@@ -2010,3 +2042,63 @@ resource "google_cloud_run_service" "cloudrun_neg" {
 `, suffix, suffix, suffix)
 }
 <% end -%>
+
+func testAccComputeBackendService_withPrivateOriginAuth(randomSuffix string) string {
+	return fmt.Sprintf(`
+resource "google_compute_backend_service" "foobar" {
+  name = "tf-test-backend-service-%s"
+  security_settings {
+    aws_v4_authentication {
+      access_key_id      = "AKIAIOSFODNN7EXAMPLE"
+      access_key         = "c4afb1cc5771d871763a393e44b703571b55cc28424d1a5e86da6ed3c154a4b9"
+      access_key_version = "prod-access-key-v1.2"
+      origin_region      = "us-east-2"
+    }
+  }
+  backend {
+    group = google_compute_global_network_endpoint_group.group.self_link
+  }
+}
+
+resource "google_compute_global_network_endpoint" "networkendpoint" {
+  global_network_endpoint_group = google_compute_global_network_endpoint_group.group.name
+  ip_address  = "8.8.8.8"
+  port        = 443
+}
+
+resource "google_compute_global_network_endpoint_group" "group" {
+  name         = "tf-test-neg-%s"
+  network_endpoint_type = "INTERNET_IP_PORT"
+}
+`, randomSuffix, randomSuffix)
+}
+
+func testAccComputeBackendService_withPrivateOriginAuthUpdate(randomSuffix string) string {
+	return fmt.Sprintf(`
+resource "google_compute_backend_service" "foobar" {
+  name = "tf-test-backend-service-%s"
+  security_settings {
```

Comment:
Although tests are missing you are still missing two fields that must be included somewhere in the tests. This is seen by the recent VCR report: https://github.com/GoogleCloudPlatform/magic-modules/pull/11042#issuecomment-2231262931

Reply:
Sorry for the delay, I needed to fix some issues with the current implementation of the 'clientTlsPolicy' field. I created a new test for these fields since they cannot be used together with the 'aws_v4_authentication' fields.